lib/config/schemas/system.config.json

```diff
@@ -0,0 +1,107 @@
+{
+  "$id": "http://express-gateway.io/models/system.config.json",
+  "type": "object",
+  "properties": {
+    "db": {
+      "type": "object",
+      "properties": {
+        "redis": {
+          "type": "object",
+          "properties": {
+            "emulate": {
+              "type": "boolean",
+              "default": "true"
+            }
+          },
+          "required": [
+            "emulate"
+          ]
+        }
+      },
+      "required": [
+        "redis"
+      ]
+    },
+    "crypto": {
+      "type": "object",
+      "properties": {
+        "cipherKey": {
+          "type": "string",
+          "default": "sensitiveKey"
+        },
+        "algorithm": {
+          "type": "string",
+          "default": "aes256"
+        },
+        "saltRounds": {
+          "type": "number",
+          "default": 10
+        }
+      },
+      "required": [
+        "cipherKey",
+        "algorithm",
+        "saltRounds"
+      ]
+    },
+    "session": {
+      "type": "object"
```

Comment:
Can we get more specific here?  We know the set of properties in here, yeah?

Reply:
as part of implementing session providers it can be that the only property we know is name of the provider

Reply:
I suggest to leave it as is

Reply:
@kevinswiber List of options is [here](https://github.com/expressjs/session#options)

I didn't enumerate them because our config file is listing only some of these and I wanted to understand more.

I'm fine enumerating all of them eventually.